src/desktop/apps/sitemaps/templates/news.jade

```diff
@@ -5,7 +5,7 @@ urlset(
 )
     for article in articles
       url
-        loc= 'https://www.artsy.net/article/' + article.get('slug')
+        loc= 'https://www.artsy.net/' + (article.get('layout') == 'news' ?  'news' : 'article') + '/' + article.get('slug')
```

Comment:
Could also use this helper from reaction -- https://github.com/artsy/reaction/blob/master/src/Components/Publishing/Constants.ts#L55

Reply:
As a side note we may want to update [Cinder](https://github.com/artsy/cinder/blob/ca0c1b6f5ea29d4be3d925f68e1c866e51314b5d/src/main/scala/net/artsy/jobs/sitemaps/ArticleSitemapJob.scala#L38-L39) to align with this.